paseos/tests/communication_window_test.py

```diff
@@ -143,6 +147,9 @@ def test_communication_link_sat_to_sat():
         central_body=earth,
     )
 
+    sat1.set_central_body_shape(Sphere([0, 0, 0], 6371000))
```

Comment:
earth_radius = 6371000

Reply:
made it a constant in test_utils instead :v: